src/Avalonia.Diagnostics/Diagnostics/ViewModels/MainViewModel.cs

```diff
@@ -271,7 +276,16 @@ public void Dispose()
 
         private void UpdateFocusedControl()
         {
-            FocusedControl = KeyboardDevice.Instance?.FocusedElement?.GetType().Name;
+            var element = KeyboardDevice.Instance?.FocusedElement;
+            FocusedControl = element?.GetType().Name;
+            _currentFocusHighlightAdorner?.Dispose();
+            if (FocusHighlighter is IBrush brush 
+                && element is InputElement input 
+                && TopLevel.GetTopLevel(input) is { } topLevel 
+                && topLevel.GetType().FullName != "Avalonia.Diagnostics.Views.MainWindow")
```

Comment:
In this project, you can simply check with "is Avalonia.Diagnostics.Views.MainWindow". Why do you need reflection?